api/v1alpha1/account_types.go

```diff
@@ -385,6 +389,15 @@ func (a *Account) IsOwnedByAccountPool() bool {
 	return false
 }
 
+func (a *Account) GetAssumeRole() string {
```

Comment:
Should I leave this function in account_types or move it to the new sts_assume_role pkg?

Reply:
I would leave it here, as it's not interacting with the aws api in any way. While at it, i would rename the function to `GetRoleARNToAssume` to make it clear that it just constructs the arn and nothing more.